graylog2-server/src/main/java/org/graylog2/rest/resources/system/GrokResource.java

```diff
@@ -104,6 +107,9 @@ public Response bulkUpdatePatterns(@ApiParam(name = "patterns", required = true)
                 throw new ValidationException("Invalid pattern " + pattern + ". Did not save any patterns.");
             }
         }
+        if (replace) {
```

Comment:
Then we have to move the pattern creation as well. Any suggestion for method names?